inc/Ui/PageDiff.php

```diff
@@ -22,17 +22,17 @@ class PageDiff extends Diff
      *
      * @param string $id  page id
      */
-    public function __construct($id = null)
+    public function __construct($id)
     {
         global $INFO;
-        $this->id = isset($id) ? $id : $INFO['id'];
+        $this->id = $id ?: $INFO['id'];
```

Comment:
Page id `0` is also a valid name, but that would be interpreted as false here.

Reply:
fixed 8792238